west.yml

```diff
@@ -91,7 +91,7 @@ manifest:
       revision: 24d84ecff195fb15c889d9046e44e4804d626c67
       path: modules/crypto/mbedtls
     - name: mcuboot
-      revision: 3fc59410b633a6d83bbb534e43aac43160f9bd32
+      revision: pull/46/head
```

Comment:
zephyrproject-rtos/mcuboot#46 is ready to merge. Wouldn't we be better to just merge this, and then reference the merged result?

Reply:
^^ Procedure is to check PR patch first for get CI results, then merge, change to the SHA and merge